libdnf/repo/Repo.cpp

```diff
@@ -861,14 +944,27 @@ std::vector<Key> Repo::Impl::retrieve(const std::string & url)
  * [2] https://bugzilla.redhat.com/show_bug.cgi?id=1769831
  * [3] https://github.com/rpm-software-management/microdnf/issues/50
  * [4] https://bugzilla.redhat.com/show_bug.cgi?id=1781601
+ * [5] https://issues.redhat.com/browse/RHEL-642
```

Comment:
Typo: invalid link, I believe it should be: https://issues.redhat.com/browse/RHEL-6421.

Reply:
Indeed.

Reply:
Fixed in 00b2d5e0fd25a15f095e4be400ad429b5002b1d3.